### ISSUE TITLE
No desktop layout is defined for Cree - Roman Orthography.

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/keyboards#205]
The keyboard layout for Cree OSK in the help.keyman.com is not defined.

[Here ](https://help.keyman.com/keyboard/fv_cree_latin_kmw/9.0/fv_cree_latin_kmw.php)is the URL to the page below.
![2018-03-20](https://user-images.githubusercontent.com/28331388/37631639-3e5012a4-2c1d-11e8-9696-361c0dfc5735.png)

**Comment by darcywong00** [comment]
This can be fixed by changing
```
<h2>Keyboard Layout</h2>
<div id='osk'>
</div>
```

to
```
<h2>Keyboard Layout</h2>
<div id='osk-tablet'>
</div>
```

**Comment by DavidLRowe** [comment]
@darcywong00 Can you point me to the file where this change needs to be made?

**Comment by darcywong00** [comment]
This file would have been release/fv/fv_cree_latin_kmw/source/help/fv_cree_latin_kmw.php

In PR #512 @LornaSIL moved it to release/fv/fv_cree_latin and already fixed the help php file.

However, the live help file doesn't show the layout 
https://help.keyman.com/keyboard/fv_cree_latin

because the build server failed to deploy version 9.1 of the keyboard. https://s.keyman.com/keyboard/fv_cree_latin/9.1/fv_cree_latin-9.1.js

@mcdurdin is busy with travels, but maybe he can  track down what happened.

**Comment by mcdurdin** [comment]
The finished version shows '1.0' because the .kps doesn't include the follow keyboard version flag, and does not include a version number either. This should be corrected, obviously...

**Comment by LornaSIL** [comment]
Fixed by PR #674
